src/osd/ReplicatedPG.cc

```diff
@@ -9161,12 +9161,12 @@ ObjectContextRef ReplicatedPG::get_object_context(const hobject_t& soid,
       pg_log.get_log().objects.find(soid)->second->op ==
       pg_log_entry_t::LOST_REVERT));
   ObjectContextRef obc = object_contexts.lookup(soid);
-  osd->logger->inc(l_osd_object_ctx_cache_total);
   if (obc) {
     osd->logger->inc(l_osd_object_ctx_cache_hit);
     dout(10) << __func__ << ": found obc in cache: " << obc
 	     << dendl;
   } else {
+    osd->logger->inc(l_osd_object_ctx_cache_total);
```

Comment:
```c++
  osd_plb.add_u64_counter(l_osd_object_ctx_cache_total, "object_ctx_cache_total", "Object context cache lookups");
```
the description reads "Object context cache lookups". so i think we should increment this counter whenever we lookup an object in `object_context` cache. well, if the description have read "Object context cache misses", your change would be sane.